dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/createfolder/CreateFolderView.java

```diff
@@ -1,17 +1,14 @@
 package com.spectralogic.dsbrowser.gui.components.createfolder;
 
 import com.airhacks.afterburner.views.FXMLView;
-import com.spectralogic.dsbrowser.gui.DeepStorageBrowserPresenter;
 import com.spectralogic.dsbrowser.gui.util.StringConstants;
 
 public class CreateFolderView extends FXMLView {
-    public CreateFolderView(final CreateFolderModel createFolderModel, final DeepStorageBrowserPresenter deepStorageBrowserPresenter) {
+    public CreateFolderView(final CreateFolderModel createFolderModel) {
         super(name -> {
```

Comment:
Do you suppose we can collapse the switch, since there is only 1 case, or do we expect that there will be more cases as we muddle through the rest of this thing?